samples/msal-angular-v2-samples/angular10-sample-app/src/app/app.module.ts

```diff
@@ -48,8 +46,7 @@ export function MSALInstanceFactory(): IPublicClientApplication {
 
 export function MSALInterceptorConfigFactory(): MsalInterceptorConfiguration {
   const protectedResourceMap = new Map<string, Array<string>>();
-  // protectedResourceMap.set('https://graph.microsoft.com/v1.0/me', ['user.read']); // Prod environment. Uncomment to use.
-  protectedResourceMap.set('https://graph.microsoft-ppe.com/v1.0/me', ['user.read']);
+  protectedResourceMap.set('https://graph.microsoft.com/v1.0/me', ['user.read']);
```

Comment:
I know this was already here, but suggest we follow the casing used in the documentation: https://learn.microsoft.com/en-us/graph/permissions-reference

Reply:
done!